README.rst

```diff
@@ -21,6 +21,8 @@ Please use the `main ZMON repository`_ to start a Vagrant demo box.
 
 Make sure the provided Vagrant-Box is up and all services are running.
 
+Update your GitHub ``client-id`` and ``client-secret`` in ``zmon-controller-app/src/main/config/application-github.yml``.
```

Comment:
> use environment variables instead

I tried to provide a description that is matching the commands below, which only set the spring profile and redis/psql URLs.

> Did you run the Vagrant Box?

I did, but I ran the build-env vm, there you don't have the env variables, see https://github.com/zalando/zmon/blob/master/build-env/vagrant/start-services.sh#L18 and https://github.com/zalando/zmon/issues/11